website/docs/cloud-docs/workspaces/settings/notifications.mdx

```diff
@@ -39,12 +39,13 @@ Go to the workspace's notifications settings and press the "Add Notification" bu
 
 Fill in the following fields to configure your notifications:
 
-* **Destination:** Terraform Cloud can deliver either a generic payload, a payload formatted specifically for Slack, or an Email. Choose which type you want from the drop-down. Payload contents are [documented below][inpage-payload].
+* **Destination:** Terraform Cloud can deliver either a generic payload, a payload formatted specifically for Slack, Microsoft Teams, or an Email. Choose which type you want from the drop-down. Payload contents are [documented below][inpage-payload].
 
 * **Name:** An arbitrary display name to use for this notification configuration.
 
-* **Webhook URL** _(only available for generic and slack webhooks):_ The destination URL to send the webhook payload to. This URL must accept HTTP or HTTPS `POST` requests, and should be able to do something useful with the chosen payload type.
+* **Webhook URL** _(only available for generic, Slack, and Microsoft Teams webhooks):_ The destination URL to send the webhook payload to. This URL must accept HTTP or HTTPS `POST` requests, and should be able to do something useful with the chosen payload type.
   * If you are sending notifications to Slack, see Slack's documentation on [how to create an incoming webhook](https://api.slack.com/messaging/webhooks#create_a_webhook).
+  * If you are sending notifications to Microsoft Teams, see the Teams documentation on [how to create an incoming webhook](https://docs.microsoft.com/en-us/microsoftteams/platform/webhooks-and-connectors/how-to/add-incoming-webhook).
 
 * **Token** _(optional; only available for generic webhooks):_ An arbitrary secret string, which Terraform Cloud will use to sign its notification webhooks. ([See "Notification Authenticity" below][inpage-hmac].) This setting is encrypted for storage, and cannot be viewed after it is saved.
```

Comment:
We always want to use "refer to" rather than "see". This is important to make sure that our content conforms to our style guide and doesn't use ableist language. 

```suggestion
  * For Slack notifications, refer to Slack's documentation on [how to create an incoming webhook](https://api.slack.com/messaging/webhooks#create_a_webhook).
  * For Microsoft Teams notifications, refer to Microsoft's documentation on [how to create an incoming webhook](https://docs.microsoft.com/en-us/microsoftteams/platform/webhooks-and-connectors/how-to/add-incoming-webhook).

* **Token** _(optional; only available for generic webhooks):_ An arbitrary secret string, which Terraform Cloud will use to sign its notification webhooks. ([See "Notification Authenticity" below][inpage-hmac].) This setting is encrypted for storage, and cannot be viewed after it is saved.
```
```suggestion
  * If you are sending notifications to Slack, see Slack's documentation on [how to create an incoming webhook](https://api.slack.com/messaging/webhooks#create_a_webhook).
  * If you are sending notifications to Microsoft Teams, see the Teams documentation on [how to create an incoming webhook](https://docs.microsoft.com/en-us/microsoftteams/platform/webhooks-and-connectors/how-to/add-incoming-webhook).

* **Token** _(optional; only available for generic webhooks):_ An arbitrary secret string, which Terraform Cloud will use to sign its notification webhooks. ([See "Notification Authenticity" below][inpage-hmac].) This setting is encrypted for storage, and cannot be viewed after it is saved.
```

Reply:
Duplicate of https://github.com/hashicorp/terraform-docs-common/pull/19#discussion_r889131844